pkg/azurefile/nodeserver.go

```diff
@@ -362,6 +395,42 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		klog.V(2).Infof("volume(%s) mount %s on %s succeeded", volumeID, source, cifsMountPath)
 	}
 
+	// If runtime OS is not windows and protocol is not nfs, save mountInfo.json
+	if runtime.GOOS != "windows" && protocol != nfs {
```

Comment:
I would use the condition `protocol == cifs` instead since that's the only protocol we've implemented.

Reply:
`protocol` could be empty, means cifs by default